assisted-inject-processor/src/main/java/com/squareup/inject/assisted/processor/AssistedInjectProcessor.kt

```diff
@@ -85,9 +85,12 @@ class AssistedInjectProcessor : AbstractProcessor() {
 
     val assistedMethods = roundEnv.findElementsAnnotatedWith<Assisted>()
         .map { it.enclosingElement as ExecutableElement }
-    // Error any non-constructor usage of @Assisted.
+    // Error any non-constructor usage of @Assisted. Methods called "copy" are also excluded due to the generated
+    // for Kotlin data classes having their parameters carry the annotations from their counterparts in the primary
+    // constructor
```

Comment:
nit:
```suggestion
    // constructor.
```

See https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html